modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,6 +1,17 @@
 :description: This page lists all changes to status codes per Neo4j version.
 = Changes to status codes per Neo4j version
 
+== Neo4j 2024.12
```

Comment:
Not sure if this is the correct release number. It's what the neo4j-PR is merging into, but I don't think we'll be releasing 2024.x? Also this is more a change between Cypher5 and Cypher25 than between 5.x and 2024.x.

Reply:
There will be no release between 5.26 LTS and 2025.1 as far as I know. So, I guess those will be new in 2025.1.

Reply:
Great, fixed :)